app/code/Magento/InventoryInStorePickup/Model/PickupLocation/Mapper/PreProcessor/FrontendName.php

```diff
@@ -25,6 +25,7 @@ class FrontendName implements PreProcessorInterface
     public function process(SourceInterface $source, $value): string
     {
         if (empty($value)) {
+            // TODO: something wrong, since by the hint 'getName' could be null, but method return only string
```

Comment:
I'd suggest to file a ticket with description. TODOs in code rarely get monitored, tracked and fixed.

Reply:
https://github.com/magento/inventory/issues/2348